README.md

```diff
@@ -97,9 +103,74 @@ class ButtonComponentStories < ViewComponent::Storybook::Stories
 end
 ```
 
+### Constructor API
```

Comment:
I would write this section a bit more directly. Introduce the concept first, active voice and avoiding 'you' etc. The examples in the ViewComponent docs like slots are good to pattern match from: https://viewcomponent.org/guide/slots.html

"Stories accept ViewComponent constructor arguments derived from Storybook controls, enabling dynamic rendering of the story.

For example:
"